docs/src/main/asciidoc/security-oidc-code-flow-authentication.adoc

```diff
@@ -10,7 +10,7 @@ include::_attributes.adoc[]
 :categories: security,web
 :toclevels: 4
 
-To protect your web applications, you can use the industry-standard authorization code flow mechanism provided by the Quarkus OpenID Connect (OIDC) extension.
+To protect your web applications, you can use the industry-standard OpenID Connect (OIDC) authorization code flow mechanism provided by the Quarkus OIDC extension.
```

Comment:
Same comment here about capitalization. 
Not urgent as after this PR is merged I will do a search and replace for any instances of inconsistent capitalization.
```suggestion
To protect your web applications, you can use the industry-standard OpenID Connect (OIDC) Authorization Code Flow mechanism provided by the Quarkus OIDC extension.
```

Reply:
Sure